js/player.js

```diff
@@ -28,20 +29,25 @@ function Player() {
   };
   this.moveRight = function() {
     this.enter(this.currentTile.tileToTheRight());
+    this.moves++;
```

Comment:
Please put that code into the enter function. It does not need to be duplicated.